src/app/data-entries/dropdown/dropdown-sample-5/dropdown-sample-5.component.ts

```diff
@@ -58,20 +50,4 @@ export class DropDownSample5Component {
             refNo: `20002`
         }]
     }];
-
-    private _positionSettings = {
-        horizontalStartPoint: HorizontalAlignment.Left,
-        verticalStartPoint: VerticalAlignment.Bottom
-    };
-    private _overlaySettings = {
-        closeOnOutsideClick: true,
-        modal: false,
-        positionStrategy: new ConnectedPositioningStrategy(this._positionSettings),
-        scrollStrategy: new CloseScrollStrategy()
```

Comment:
We had here Close scroll strategy and in the new sample it will be Absolute scroll strategy!

Reply:
Thanks for pointing this out! I don't think the scroll strategy is really relevant here though, no need to clutter up the example with custom settings.